code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1212,12 +1212,22 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	if(target.check_block())
 		target.visible_message("<span class='warning'>[target] blocks [user]'s grab attempt!</span>", \
 							"<span class='userdanger'>You block [user]'s grab attempt!</span>")
-		return 0
+		return FALSE
 	if(attacker_style && attacker_style.grab_act(user,target))
-		return 1
+		return TRUE
 	else
+		//Steal them shoes
+		if(!(target.mobility_flags & MOBILITY_STAND) && (user.zone_selected == BODY_ZONE_PRECISE_L_FOOT || user.zone_selected == BODY_ZONE_PRECISE_R_FOOT) && user.a_intent == INTENT_GRAB && target.shoes)
```

Comment:
Can you even target L_FOOT and R_FOOT?
Also the intent check isn't needed, this is the grab proc, guess when this is called.
Also shoes needs an istype check since someone thought it was a good idea that you could equip any item into any inventory slot.

Reply:
Can't you grab in other intents with ctrl+click?

Reply:
I concede that point